extensions/scarb-cairo-run/tests/arguments.rs

```diff
@@ -0,0 +1,106 @@
+use assert_fs::TempDir;
+use indoc::indoc;
+use scarb_test_support::project_builder::ProjectBuilder;
+use snapbox::cmd::{cargo_bin, Command};
+
+fn setup_fib_three_felt_args(t: &TempDir) {
+    ProjectBuilder::start()
+        .name("hello")
+        .version("0.1.0")
+        .lib_cairo(indoc! {r#"
+        fn main(a: felt252, b: felt252, n: felt252) -> felt252 {
+            fib(a, b, n)
+        }
+
+        fn fib(a: felt252, b: felt252, n: felt252) -> felt252 {
+            match n {
+                0 => a,
+                _ => fib(b, a + b, n - 1),
+            }
+        }
+        "#})
+        .build(t);
+}
+
+#[test]
+fn valid_number_of_args() {
+    let t = TempDir::new().unwrap();
+    setup_fib_three_felt_args(&t);
+
+    Command::new(cargo_bin("scarb"))
+        .env("SCARB_TARGET_DIR", t.path())
```

Comment:
I don't think you need it here. It has been added to tests that run on projects from `examples` directory, as in that case the `target` dir is shared by all the tests (e.g. `examples/hello_world/target` - this path does not change at all!). Here you setup a temp dir for a test project each time, so I think you should be good to go. 
While at it - would you mind adding a comment in the other tests file explaining why has this been done?